FILES/N_0x10144267dd22866c.md

```diff
@@ -1,29 +1,22 @@
 ---
 ns: FILES
+aliases: ["_0x10144267DD22866C"]
 ---
-## _0x10144267DD22866C
+## GET_TATTOO_SHOP_DLC_ITEM_INDEX
 
 ```c
 // 0x10144267DD22866C
-int _0x10144267DD22866C(Hash overlayHash, Any p1, int character);
-```
-
-Returns some sort of index/offset for overlays/decorations.
-
-```
-Character types:
-0 = Michael,
-1 = Franklin,
-2 = Trevor,
-3 = MPMale,
-4 = MPFemale
+int GET_TATTOO_SHOP_DLC_ITEM_INDEX(cs_type(Hash) int character, cs_type(Any) int collection, cs_type(int) Hash preset);
 ```
 
 ```
 NativeDB Introduced: v2189
 ```
 
 ## Parameters
-* **overlayHash**:
-* **p1**:
-* **character**:
+* **character**: AKA eFaction in shop_tattoo.meta. Which character this tattoo belongs to. See [GET_NUM_TATTOO_SHOP_DLC_ITEMS](#_0x278F76C3B0A8F109).
+* **collection**: Which collection this tattoo belongs to.
```

Comment:
You can add that `-1` will check every collections too.

Reply:
Thank u! Done :)